app/views/timeline_entries/_form.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= render "govuk_publishing_components/components/input", {
+  label: {
+    text: "Enter the heading of the timeline entry",
+    bold: true,
+  },
+  name: "timeline_entry[heading]",
+  value: @timeline_entry.heading,
+  id: "heading",
+  error_message: error_items(@timeline_entry.errors.messages, :heading),
+} %>
+
+<%= render "components/markdown_editor", {
+  label: {
+    text: "Content",
+    bold: true
+  },
+  textarea: {
+    name: "timeline_entry[content]",
+    id: "content",
+    rows: 20,
+    value: @timeline_entry.content
+  },
+  heading_buttons: true,
+  blockquote_button: true,
+  numbered_list_button: true,
+  bullets_button: true
+} %>
```

Comment:
It looks like this is missing the error display that is shown for the heading field.

For example: 

![Screenshot 2021-01-12 at 11 06 25](https://user-images.githubusercontent.com/282717/104306764-40a8e380-54c6-11eb-8d73-2d95a57847ad.png)

Reply:
Ah! It looks like this is missing in the component. I'll add it.